assets/js/util/scroll.js

```diff
@@ -58,11 +50,36 @@ export function getContextScrollTop( context, breakpoint ) {
 	const anchorAdjustment =
 		context === '#googlesitekit-pagespeed-header' ? 80 : 0;
 
-	return (
-		contextTop +
-		global.scrollY -
-		headerHeight -
-		navigationHeight -
-		anchorAdjustment
+	return contextTop + global.scrollY - headerHeight - anchorAdjustment;
+}
+
+/**
+ * Gets the height of the sticky header.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} breakpoint The current breakpoint.
+ * @return {number} The height of the sticky header.
+ */
+export function getHeaderHeight( breakpoint ) {
+	let headerHeight = 0;
+
+	const header = document.querySelector( '.googlesitekit-header' );
+	if ( header ) {
+		headerHeight +=
```

Comment:
Very minor nitpick, but this doesn't need to be `+=`, because before this `headerHeight` is always `0`.

It's minor, but seeing the `+=` made me look for where else `headerHeight` was modified.

I guess it's more flexible so let's leave it, but it surprised me.

Reply:
Updated.